test/convert.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+
+const test = require('ava');
+const uniqueString = require('unique-string');
+const fs = require('fs');
+const sinon = require('sinon');
+const path = require('path');
+
+const {getVideoMetadata} = require('./helpers/video-utils');
+const {almostEquals} = require('./helpers/assertions');
+
+const getRandomFileName = (ext = 'mp4') => `${uniqueString()}.${ext}`;
+
+const input = path.resolve(__dirname, 'fixtures', 'input.mp4');
+const retinaInput = path.resolve(__dirname, 'fixtures', 'input@2x.mp4');
+
+const {mockImport} = require('./helpers/mocks');
+
+mockImport('./common/analytics', 'analytics');
+mockImport('./service-context', 'service-context');
+
+const {convertTo} = require('../main/convert');
+
+test.afterEach.always(t => {
+  if (t.context.outputPath && fs.existsSync(t.context.outputPath)) {
+    fs.unlink(t.context.outputPath, () => {});
+  }
+});
```

Comment:
```suggestion
test.afterEach.always(async t => {
  if (t.context.outputPath && fs.existsSync(t.context.outputPath)) {
    await fs.promises.unlink(t.context.outputPath);
  }
});
```

Or maybe just use the sync function, since it's unlikely to have much overhead and sync is usually faster.